Build/Scripts/BuildLinux.js

```diff
@@ -23,6 +23,9 @@ function copyAtomicNET() {
 
 function copyAtomicEditor() {
 
+    if ( config["noclean"])
+        host.cleanCreateDir(editorAppFolder);
```

Comment:
Ok, I see,  fs,copySync should take a clobber option which will overwrite: https://github.com/jprichardson/node-fs-extra#copy though it would be nice to not have to specify that every call :/

I think we should change all the builds to clean the editor app folder (and I think some checks needs to be put in for noclean + doc/example generation), so let's move this folder creation down to where the host.setupDirs is called, removing it from the provided lists to that and do a clean create regardless... will follow up with other builds